tests/base/src/test/java/org/wildfly/security/http/impl/AbstractBaseHttpTest.java

```diff
@@ -92,14 +94,16 @@
 
 import mockit.Mock;
 import mockit.MockUp;
+import org.wildfly.security.x500.cert.SelfSignedX509CertificateAndSigningKey;
```

Comment:
Just minor, this should be moved with the other org.wildfly.security imports.

Reply:
@fjuma Done, moved it with the other security imports.